persistence/scaladsl/src/test/scala/com/lightbend/lagom/scaladsl/persistence/AbstractReadSideSpec.scala

```diff
@@ -61,28 +60,37 @@ trait AbstractReadSideSpec extends ImplicitSender with ScalaFutures with Eventua
     )
   }
 
+  class AlwaysReplyDone extends Actor {
+    def receive = {
+      case Execute =>
+
+        processorFactory()
+          .buildHandler
+          .globalPrepare()
+          .map { _ => Done } pipeTo sender()
+
+    }
+  }
+
   private def createReadSideProcessor() = {
     /* read side and injector only needed for deprecated register method */
-    val readSide = system.actorOf(
-      ReadSideActor.props[TestEntity.Evt](
-        processorFactory,
-        eventStream,
-        classOf[TestEntity.Evt],
-        new ClusterStartupTask(testActor),
-        20.seconds
-      )
-    )
 
-    readSide ! EnsureActive(tag.tag)
+    val processorProps = ReadSideActor.props[TestEntity.Evt](
+      processorFactory,
+      eventStream,
+      classOf[TestEntity.Evt],
+      new ClusterStartupTask(system.actorOf(Props(new AlwaysReplyDone))),
+      3.seconds
+    )
 
-    expectMsg(Execute)
+    val readSide: ActorRef =
+      system.actorOf(
+        BackoffSupervisor.propsWithSupervisorStrategy(
+          processorProps, "processor", 500.milliseconds, 1.second, 0.2, SupervisorStrategy.stoppingStrategy
+        )
+      )
 
-    processorFactory()
-      .buildHandler
-      .globalPrepare()
-      .foreach { _ =>
-        readSide ! Done
-      }
+    system.scheduler.schedule(Duration.Zero, 1.second, readSide, EnsureActive(tag.tag))
```

Comment:
For `JdbcReadSideSpec` to work under Travis with latest AP-jdbc (which includes the very restrictive 100ms timeout) it is necessary to setup the actor under test so that it can be restarted. That means adding a BackoffSupervisor, sending `EnsureActive` constantly and providing a GlobalStartupTask that's fully compliant (given an `Execute`, respond a `Done`)  .